cmd/core-metadata/main.go

```diff
@@ -48,16 +51,25 @@ func main() {
 	flag.Usage = usage.HelpCallback
 	flag.Parse()
 
-	httpServer := httpserver.NewBootstrap(metadata.LoadRestRoutes())
+	configuration := &metadataConfig.ConfigurationStruct{}
```

Comment:
Premature.  Remove and reference the existing metadata.Configuration global where configuration is actually referenced/used in the code that follows.